### DISCUSSION TITLE
Benchmark tests failing sometimes

### DISCUSSION THREAD

**Discussion by XSpielinbox** [focus-shift/jollyday#510] [Q&A] [answered]
Hello,

I've now received an email about a failed action in my fork for the second time in two days again.
The Benchmarks can succeed sometimes: [example from yesterday](https://github.com/XSpielinbox/jollyday/actions/runs/8669895226/job/23777239469), but sometimes they don't: [example from today](https://github.com/XSpielinbox/jollyday/actions/runs/8677719308/job/23793708629), though they fail not by much as far as I understand it.

I cannot reproduce these failures on my local machine and in this repo these errors also don't seem to occur: https://github.com/focus-shift/jollyday/actions

Do you have any Idea, what could cause these issues and how to solve them?

**Comment by derTobsch** [comment, marked as answer]
Each benchmark test comes with a minimum ops/s (operations per second) requirement that it needs to meet to pass. Usually, these tests run fine on GitHub Actions since the ops/s benchmarks are based on what's considered normal performance for those actions. But every now and then, if GitHub Actions are having a rough time or if there's a lot going on with them, they might fail to meet those requirements. It doesn't happen a lot, but it's something to keep in mind.
